pyseir/cdc/output_mapper.py

```diff
@@ -0,0 +1,526 @@
+
+    def run_model_ensemble(self, override_param_names=['R0', 'I_initial', 'E_initial', 'suppression_policy']):
+        """
+        Get model ensemble by running models under different parameter sets
+        sampled from parameter prior distributions.
+
+        Parameters
+        ----------
+        override_param_names: list(str)
+            Names of model parameters to override. Default list include the
+            parameters varied for MLE inference.
+
+        Returns
+        -------
+        model_ensemble: list(SEIRModel)
+            List of SEIR models ran under parameter sets randomly generated
+            from the parameter prior distributions.
+        """
+
+        override_params = {k: v for k, v in self.model.__dict__.items() if k in override_param_names}
+        er = EnsembleRunner(fips=self.fips)
+        model_ensemble = er._model_ensemble(override_params=override_params, N_samples=self.N_samples)
+        return model_ensemble
+
+    def forecast_target(self, model, target, unit):
+        """
+        Runs forecast of a target with given model.
+
+        Parameters
+        ----------
+        model: SEIRModel
+            SEIR model to run the forecast.
+        target: Target
+            The target to forecast.
+        unit: ForecastTimeUnit
+            Time unit to aggregate the forecast. Currently supports
+            ForecastTimeUnit.DAY
+
+        Returns
+        -------
+        target_forecast: np.array
+            Forecast of target at given unit (currently only supports daily
+            forecast), with shape (len(self.forecast_time_range),)
+        """
+        if unit is ForecastTimeUnit.DAY:
+            if target is Target.INC_DEATH:
+                target_forecast = self.forecast_given_time_range(model.results['total_deaths_per_day'], model.t_list)
+
+            elif target is Target.INC_HOSP:
+                target_forecast = self.forecast_given_time_range(np.append([0],
+                                                                 np.diff(model.results['HGen_cumulative']
+                                                                       + model.results['HICU_cumulative'])),
+                                                                 model.t_list)
+
+            elif target is Target.CUM_DEATH:
+                target_forecast = self.forecast_given_time_range(model.results['D'], model.t_list)
+
+            else:
+                raise ValueError(f'Target {target} is not implemented')
+
+            target_forecast = pd.Series(target_forecast,
+                                        index=target_column_name([(t - self.forecast_date).days for t in
+                                                                  self.forecast_time_range],
+                                                                  target, unit))
+
+            return target_forecast
+        else:
+            raise ValueError(f'Forecast time unit {unit} is not supported')
+
+
```

Comment:
Will be more general in the future:

```suggestion
        else:
            raise ValueError(f'Forecast time unit {unit} is not supported')

        return target_forecast
```

```